content/docs/1.6.0/volumes-and-nodes/maintenance.md

```diff
@@ -12,91 +12,229 @@ This section describes how to handle planned node maintenance or upgrading Kuber
 - [Upgrading Kubernetes](#upgrading-kubernetes)
   - [In-place Upgrade](#in-place-upgrade)
   - [Managed Kubernetes](#managed-kubernetes)
+- [Node Drain Policy Recommendations](#node-drain-policy-recommendations)
 
 ## Updating the Node OS or Container Runtime
 
 1. Cordon the node. Longhorn will automatically disable the node scheduling when a Kubernetes node is cordoned.
 
 1. Drain the node to move the workload to somewhere else.
 
-    You will need to use `--ignore-daemonsets` to drain the node.
-    The `--ignore-daemonsets` is needed because Longhorn deployed some daemonsets such as `Longhorn manager`, `Longhorn CSI plugin`, `engine image`.
+   You will need to use `--ignore-daemonsets` to drain the node. The `--ignore-daemonsets` is needed because Longhorn
+   deployed some daemonsets such as `Longhorn manager`, `Longhorn CSI plugin`, `engine image`.
 
-    The running replicas on the node will be stopped at this stage. They will be shown as `Failed`.
+   While the drain proceeds, engine processes on the node will be migrated with the workload pods to other nodes.
 
-   > **Note:**
-   > By default, if there is one last healthy replica for a volume on
-   > the node, Longhorn will prevent the node from completing the drain
-   > operation, to protect the last replica and prevent the disruption of the
-   > workload. You can control this behavior in the setting [Node Drain Policy](../../references/settings#node-drain-policy), or [evict
-   > the replica to other nodes before draining](../disks-or-nodes-eviction).
+   > **Note:** For volumes that are not attached through the CSI flow on the node (for example, manually attached using
```

Comment:
~For~ Volumes that are not attached ... 
~they~ will not be automatically attached ...

Reply:
Fixing in an amend.